site/.vuepress/enhanceApp.js

```diff
@@ -0,0 +1,163 @@
+    '/blog/2017-12-04/openml/',
+    '/blog/2017-09-28/zegami/',
+    '/blog/2017-05-23/cmso/',
+    '/blog/2017-08-09/collections-as-data/',
+    '/blog/2017-05-24/the-data-retriever/',
+    '/blog/2016-11-15/dataship/',
+    '/blog/2017-04-11/dataworld/',
+    '/blog/2017-03-28/john-snow-labs/',
+    '/blog/2016-11-15/tesera/',
+    '/blog/2016-11-15/open-power-system-data/',
+    '/blog/2018-07-16/oleg-lavrovsky/',
+    '/blog/2017-10-26/matt-thompson/',
+    '/blog/2017-10-24/georges-labreche/',
+    '/blog/2017-10-27/open-knowledge-greece/',
+    '/blog/2018-07-16/ori-hoch/',
+    '/blog/2017-11-01/daniel-fireman/',
+    '/blog/2019-07-03/nes/',
+    '/blog/2020-01-23/nes-tool/',
+    '/blog/2019-09-12/andre-heughebaert/',
+    '/blog/2019-07-02/stephan-max/',
+    '/blog/2019-07-09/open-referral/',
+    '/blog/2020-01-22/frictionless-darwincore/',
+    '/blog/2020-01-22/open-referral-tool/',
+    '/blog/2017-12-19/dm4t/',
+    '/blog/2017-12-12/ukds/',
+    '/blog/2017-12-15/university-of-pittsburgh/',
+    '/blog/2017-10-24/elife/',
+    '/blog/2017-08-15/causa-natura-pescando-datos/',
+    '/blog/2017-08-15/university-of-cambridge/',
+    '/blog/2018-05-07/pacific-northwest-national-laboratory-active-data-biology/',
+    '/blog/2018-05-07/using-data-packages-in-clojure/',
+    '/blog/2018-04-28/using-data-packages-in-java/',
+    '/blog/2018-02-16/using-data-packages-in-go/',
+    '/blog/2018-02-14/creating-tabular-data-packages-in-r/',
+    '/blog/2018-02-14/using-data-packages-in-r/',
+    '/blog/2016-08-29/using-data-packages-in-python/',
+    '/blog/2016-07-21/creating-tabular-data-packages-in-python/',
+    '/blog/2018-04-04/creating-tabular-data-packages-in-javascript/',
+    '/blog/2018-04-06/joining-data-in-python/',
+    '/blog/2018-04-05/joining-tabular-data-in-python/',
+    '/blog/2018-07-16/publish-data-as-data-packages/',
+    '/blog/2018-03-27/applying-licenses/',
+    '/blog/2016-08-30/publish/',
+    '/blog/2018-08-29/publish-online/',
+    '/blog/2016-07-21/publish-any/',
+    '/blog/2016-07-21/publish-tabular/',
+    '/blog/2016-04-30/publish-geo/',
+    '/blog/2018-07-16/publish-faq/',
+    '/blog/2018-07-16/point-location-data/',
+    '/blog/2018-07-09/csv/',
+    '/blog/2018-07-09/validating-data/',
+    '/blog/2018-03-07/well-packaged-datasets/',
+    '/blog/2018-07-16/visible-findable-shareable-data/',
+    '/blog/2018-07-16/validated-tabular-data/',
+    '/blog/2019-05-20/used-and-useful-data/',
+    '/blog/2018-03-12/automatically-validated-tabular-data/',
+    '/blog/2018-03-12/data-publication-workflow-example/',
+    '/reproducible-research/',
+    '/contribute',
+    '/contribute',
+    '/data-packages/',
+    '/data-packages/',
+    '/table-schema/',
+    '/data-packages/',
+    '/data-packages/guide/',
+    ];
+
+let redirectList=[]
+
+for (let i = 0; i < ((oldUrl.length)); i++) {
+    redirectList[i] = {path: oldUrl[i], redirect: destinationUrl[i]}
+}
+console.log(redirectList)
```

Comment:
You probably don't want to `console.log` this in usual operations. Can you remove it?